Makefile

```diff
@@ -154,7 +154,7 @@ test-acceptance: build-platform-assets
 	py.test tests/acceptance --cov . --cov-report="xml:coverage.xml" --junit-xml="junit.xml" --html="pytest.html"
 	@echo ""
 
-lint: lint-python lint-js
+lint: lint-python lint-js measure-js-bundle
```

Comment:
Is this going to work? Doesn't lint get run before test (which calls webpack)?

Reply:
i think you're right, good catch 😬